### DISCUSSION TITLE
What does "Alternative launch" do?

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [U-C-S/Hurl#130] [Q&A]
I see it in the code, but I simply have no idea what it does. I have tried to install a different browser, like Chrome Canary, and use a profile to test, but I see nothing in the interface...

**Comment by U-C-S** [comment]
https://github.com/U-C-S/Hurl/wiki/Editing-UserSettings.json#alternatelaunches

The name "Alternate Launch" is bit outdated, could probably need some "alternative" names that better match it's functionality. (Same with 'Timed Default')
